infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -124,10 +124,24 @@ rm -rf $WORK/llvm-stage1 $WORK/llvm-stage2
 # Use the clang we just built from now on.
 CMAKE_EXTRA_ARGS="-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++"
 
+function cmake_libcxx {
+  extra_args="$@"
+  cmake -G "Ninja" \
+      -DLIBCXX_ENABLE_SHARED=OFF \
+      -DLIBCXX_ENABLE_STATIC_ABI_LIBRARY=ON \
+      -DLIBCXXABI_ENABLE_SHARED=OFF \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DLLVM_TARGETS_TO_BUILD="$TARGET_TO_BUILD" \
+      -DLLVM_ENABLE_PROJECTS="libcxx;libcxxabi" \
+      -DLLVM_BINUTILS_INCDIR="/usr/include/" \
+      $extra_args \
+      $LLVM_SRC/llvm
+}
+
 # 32-bit libraries.
 mkdir -p $WORK/i386
 cd $WORK/i386
-cmake_llvm $CMAKE_EXTRA_ARGS \
+cmake_libcxx $CMAKE_EXTRA_ARGS \
```

Comment:
This change was necessary because enabling projects compiler-rt when we tried to build the msanified libcxx resulted in the error below (CC @morehouse @kcc @hctim)
Luckily, it shouldn't really be necessary to enable those projects when building libcxx.
```
CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/msan/tests/CMakeLists.txt:138 (add_custom_libcxx)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/msan/tests/CMakeLists.txt:138 (add_custom_libcxx)


CMake Error at /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390:EVAL:2 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390 (cmake_language)
  /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:658 (ExternalProject_Add_Step)
  /src/llvm-project/compiler-rt/lib/msan/tests/CMakeLists.txt:138 (add_custom_libcxx)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/tsan/CMakeLists.txt:282 (add_custom_libcxx)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/tsan/CMakeLists.txt:282 (add_custom_libcxx)


CMake Error at /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390:EVAL:2 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390 (cmake_language)
  /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:658 (ExternalProject_Add_Step)
  /src/llvm-project/compiler-rt/lib/tsan/CMakeLists.txt:282 (add_custom_libcxx)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-all" does not exist.
Call Stack (most recent call first):
  CMakeLists.txt:1051 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-all" does
  not exist.
Call Stack (most recent call first):
  CMakeLists.txt:1051 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-builtins" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/builtins/CMakeLists.txt:90 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-builtins"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/builtins/CMakeLists.txt:90 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-interception" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/interception/CMakeLists.txt:14 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-interception" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/interception/CMakeLists.txt:14 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-lsan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/lsan/CMakeLists.txt:36 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-lsan" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/lsan/CMakeLists.txt:36 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-ubsan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/ubsan/CMakeLists.txt:138 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-ubsan" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/ubsan/CMakeLists.txt:138 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-cfi" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/cfi/CMakeLists.txt:96 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-cfi" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/cfi/CMakeLists.txt:96 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-cfi-and-supported" does not
  exist.
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/test/cfi/CMakeLists.txt:100 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-cfi-and-supported" does not exist.
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/test/cfi/CMakeLists.txt:100 (add_lit_target)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-sanitizer" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/sanitizer_common/CMakeLists.txt:111 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-sanitizer"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/sanitizer_common/CMakeLists.txt:111 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-fuzzer-default-x86_64" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:69 (add_lit_testsuite)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:82 (test_fuzzer)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-fuzzer-default-x86_64" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:69 (add_lit_testsuite)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:82 (test_fuzzer)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-fuzzer-static-libcxx-x86_64"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:69 (add_lit_testsuite)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:88 (test_fuzzer)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-fuzzer-static-libcxx-x86_64" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:69 (add_lit_testsuite)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:88 (test_fuzzer)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-fuzzer-default-i386" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:69 (add_lit_testsuite)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:82 (test_fuzzer)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-fuzzer-default-i386" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:69 (add_lit_testsuite)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:82 (test_fuzzer)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-fuzzer-unit" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:32 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-fuzzer-unit" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:32 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-fuzzer-static-libcxx-i386"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:69 (add_lit_testsuite)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:88 (test_fuzzer)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-fuzzer-static-libcxx-i386" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:69 (add_lit_testsuite)
  /src/llvm-project/compiler-rt/test/fuzzer/CMakeLists.txt:88 (test_fuzzer)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-asan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/asan/CMakeLists.txt:157 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-asan" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/asan/CMakeLists.txt:157 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-asan-dynamic" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/asan/CMakeLists.txt:164 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-asan-dynamic" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/asan/CMakeLists.txt:164 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-dfsan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/dfsan/CMakeLists.txt:28 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-dfsan" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/dfsan/CMakeLists.txt:28 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-msan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/msan/CMakeLists.txt:58 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-msan" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/msan/CMakeLists.txt:58 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-hwasan-lam" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/hwasan/CMakeLists.txt:38 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-hwasan-lam"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/hwasan/CMakeLists.txt:38 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-hwasan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/hwasan/CMakeLists.txt:31 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-hwasan"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/hwasan/CMakeLists.txt:31 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-tsan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/tsan/CMakeLists.txt:105 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-tsan" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/tsan/CMakeLists.txt:105 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-safestack" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/safestack/CMakeLists.txt:28 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-safestack"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/safestack/CMakeLists.txt:28 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-scudo" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/scudo/CMakeLists.txt:32 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-scudo" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/scudo/CMakeLists.txt:32 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-ubsan-minimal" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/ubsan_minimal/CMakeLists.txt:23 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-ubsan-minimal" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/ubsan_minimal/CMakeLists.txt:23 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-gwp_asan" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/gwp_asan/CMakeLists.txt:40 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-gwp_asan"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/gwp_asan/CMakeLists.txt:40 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-profile" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/profile/CMakeLists.txt:29 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-profile"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/profile/CMakeLists.txt:29 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-memprof" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/memprof/CMakeLists.txt:49 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-memprof"
  does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/memprof/CMakeLists.txt:49 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-memprof-dynamic" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/memprof/CMakeLists.txt:54 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-memprof-dynamic" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/memprof/CMakeLists.txt:54 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-xray" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/xray/CMakeLists.txt:30 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-xray" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/xray/CMakeLists.txt:30 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-orc" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/orc/CMakeLists.txt:22 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-orc" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/orc/CMakeLists.txt:22 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-crt" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/crt/CMakeLists.txt:31 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target "check-crt" does
  not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/crt/CMakeLists.txt:31 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang" of target "check-shadowcallstack" does not
  exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/shadowcallstack/CMakeLists.txt:18 (add_lit_testsuite)


CMake Error at cmake/modules/AddLLVM.cmake:1808 (add_dependencies):
  The dependency target "clang-resource-headers" of target
  "check-shadowcallstack" does not exist.
Call Stack (most recent call first):
  cmake/modules/AddLLVM.cmake:1829 (add_lit_target)
  /src/llvm-project/compiler-rt/test/shadowcallstack/CMakeLists.txt:18 (add_lit_testsuite)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/msan/tests/CMakeLists.txt:138 (add_custom_libcxx)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/msan/tests/CMakeLists.txt:138 (add_custom_libcxx)


CMake Error at /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390:EVAL:2 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390 (cmake_language)
  /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:658 (ExternalProject_Add_Step)
  /src/llvm-project/compiler-rt/lib/msan/tests/CMakeLists.txt:138 (add_custom_libcxx)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/msan/tests/CMakeLists.txt:138 (add_custom_libcxx)


CMake Error at /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390:EVAL:2 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390 (cmake_language)
  /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:658 (ExternalProject_Add_Step)
  /src/llvm-project/compiler-rt/lib/msan/tests/CMakeLists.txt:138 (add_custom_libcxx)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/tsan/CMakeLists.txt:282 (add_custom_libcxx)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/tsan/CMakeLists.txt:282 (add_custom_libcxx)


CMake Error at /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390:EVAL:2 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390 (cmake_language)
  /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:658 (ExternalProject_Add_Step)
  /src/llvm-project/compiler-rt/lib/tsan/CMakeLists.txt:282 (add_custom_libcxx)


CMake Error at /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:573 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /src/llvm-project/compiler-rt/lib/tsan/CMakeLists.txt:282 (add_custom_libcxx)


CMake Error at /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390:EVAL:2 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:clang>

  No target "clang"
Call Stack (most recent call first):
  /usr/local/share/cmake-3.21/Modules/ExternalProject.cmake:2390 (cmake_language)
  /src/llvm-project/compiler-rt/cmake/Modules/AddCompilerRT.cmake:658 (ExternalProject_Add_Step)
  /src/llvm-project/compiler-rt/lib/tsan/CMakeLists.txt:282 (add_custom_libcxx)


-- Generating done
```